server/routes/countries.js

```diff
@@ -0,0 +1,85 @@
+const express = require('express');
+const router = express.Router();
+const api = require('./countries.js');
+
+// basic countries objects
+var countries = [
+    {id: 0, name: "Canada", abbrev: "CA"},
+    {id: 1, name: "Germany", abbrev: "DE"},
+    {id: 2, name: "France", abbrev: "FR"},
+    {id: 3, name: "Great Britain", abbrev: "GB"},
+    {id: 4, name: "India", abbrev: "IN"},
+    {id: 5, name: "Japan", abbrev: "JP"},
+    {id: 6, name: "Korea", abbrev: "KR"},
+    {id: 7, name: "Mexico", abbrev: "MX"},
+    {id: 8, name: "Russia", abbrev: "RU"},
+    {id: 9, name: "United States", abbrev: "US"},
+];
```

Comment:
Curious - what are the ids for? Are they meant to map to the categories results?